<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeWidget" name="DBTreeView">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>256</width>
      <height>361</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QTreeWidget" name="CachedTreeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>361</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="UploadToCashBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NewItemBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteItemBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetValueItemBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ApplyBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <zorder>CachedTreeView</zorder>
   <zorder>UploadToCashBtn</zorder>
   <zorder>NewItemBtn</zorder>
   <zorder>DeleteItemBtn</zorder>
   <zorder>SetValueItemBtn</zorder>
   <zorder>ApplyBtn</zorder>
   <zorder>ResetBtn</zorder>
   <zorder>DBTreeView</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
